### PR TITLE
Feature/remove to console from modules

### PR DESCRIPTION
**Pull request by EluciusFTW**
So, I started removing toConsole from the first modules. I also extended the panel to now accept renderable title and content (before it only worked with unstyled strings).

### CODE REVIEW

File: src/spectrecoff-cli/commands/Output.fs [hunk 1]
```diff
@@ -81,7 +81,7 @@ type OutputExample() =
                 P "several"
                 E "items"
             ]
-            NewLine
+            SpectreCoff.Rule.rule "Links  and Emojis"
```
Comment (DanielMuckelbauer) [inline src/spectrecoff-cli/commands/Output.fs:84]: Extra whitespace here 🤷‍♂️

File: src/spectrecoff-cli/commands/Panel.fs [hunk 2]
```diff
@@ -12,11 +13,18 @@ type PanelExample() =
...
     override _.Execute(_context, _) = 
-        let examplePanel = panel "First !" "Important text can be highlighted by surrounding it with a border and title." 
-        examplePanel |> toConsole    
+        let principles =  BI [
+            C "Composability over Inheritance" 
+            P "Make illegal states irrepresentable"
+            P "Types are your friend"
+            C "Immutability over, well, mutability"
+            C "Declarative over Imperative"
+        ]
```
Comment (DanielMuckelbauer) [inline src/spectrecoff-cli/commands/Panel.fs:22]: Spreading some wisdom

File: src/spectrecoff-cli/commands/Progress.fs [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 open Spectre.Console
 open Spectre.Console.Cli
-open SpectreCoff
```
Comment (DanielMuckelbauer) [inline src/spectrecoff-cli/commands/Progress.fs:5]: Too much space

**Review by DanielMuckelbauer — APPROVED** [review]

**Comment by DanielMuckelbauer** [comment]
Love it

File: src/spectrecoff-cli/commands/Progress.fs (hunk 3 above)
Comment (EluciusFTW) [reply src/spectrecoff-cli/commands/Progress.fs:5]: Don't see what you mean. The blank line between Spectre and SpectreCoff opens? That was on purpose.

File: src/spectrecoff-cli/commands/Output.fs (hunk 1 above)
Comment (EluciusFTW) [reply src/spectrecoff-cli/commands/Output.fs]: Gone now.

**Pull request merged**